keeper/keeper.go

```diff
@@ -122,3 +132,47 @@ func (k Keeper) IsSenderValidator(ctx context.Context, sender string, expectedVa
 func (k Keeper) Logger() log.Logger {
 	return k.logger
 }
+
+// updateBondedPoolPower updates the bonded pool to the correct power for the network.
+func (k Keeper) UpdateBondedPoolPower(ctx context.Context) error {
```

Comment:
I probably call this function more than needed. Only is called on a change to the validator set, so see this as fine given sets will be mostly static